README.md

```diff
@@ -12,6 +12,8 @@ Use the package manager [pip](https://pip.pypa.io/en/stable/) to install `safegr
 pip install -q --upgrade git+https://github.com/SafeGraphInc/safegraph_py
```

Comment:
At the top of the Readme, let's add a brief explanation of the differences between the `safegraph_py_functions` part of this library and your new `cbg_functions` stuff.  Right now it's a bit disjointed and confusing what this package is. so sgpy is for working with SafeGraph patterns data, cbg_functions is for accessing safegraph's hosted copy of the 2016-2019 ACS data from the US Census. An explanation like that.